Jenkinsfile.e2e.vms

```diff
@@ -0,0 +1,51 @@
+pipeline {
+  agent {
+    node {
+      label 'bare-metal'
+    }
+  }
+  environment {
+    BUILD_USER = 'clear'
+    CI_ID = "${env.BUILD_TAG}"
+    KUBECONFIG = "${env.HOME}/.kube/config-${env.BUILD_TAG}"
+  }
+  stages {
+    stage('Build Images') {
+      steps {
+        sh "sed -i 's|canary|$CI_ID|' Makefile"
+        sh 'docker run --rm -v $PWD:/go/src/github.com/intel/pmem-csi -v /var/run/docker.sock:/var/run/docker.sock pmem-csi-ci:go-alpine make build-images'
+      }
+    }
+    stage('Setup K8s') {
+      steps {
+	sh '$HOME/dev/k8s/ci-scripts/deploy-k8s.sh $CI_ID $HOME/vms/images/clear-latest-cloud.img  $BUILD_USER'
```

Comment:
Please don't use deploy-k8s.sh.

We need to verify that the cluster start method which is used by everyone else (`make start`) works.